tests/openvino/datasets_helpers.py

```diff
@@ -129,3 +129,10 @@ def transform_fn(data_item):
 
     calibration_dataset = Dataset(model_evaluator.dataset, transform_fn)
     return calibration_dataset
+
+
+def test_dataset_length():
```

Comment:
As your PR makes changes in a common part, I would suggest adding a test on this functionality in a common part as well.

Reply:
Thanks. I moved this test to common and also added other Dataset-related tests because didn't find any other tests for it 🤔